packages/trader/src/Modules/Trading/Components/Form/TradeParams/amount-mobile.jsx

```diff
@@ -80,7 +80,7 @@ const Basis = observer(
                     {(is_turbos || is_vanilla) && <MinMaxStakeInfo />}
                     <div
                         className={classNames('trade-params__amount-keypad', {
-                            strike__pos: contract_type === 'vanilla',
+                            strike__pos: contract_type === VANILLALONG.CALL || contract_type === VANILLALONG.PUT,
```

Comment:
@kate-deriv  will `isVanillaContract(contract_type)` or `is_vanilla` work here? :)

Reply:
Both are working; I'll pick up option `is_vanilla` as it's already used above. Thank you for noticing